src/host/polyfills/simpleView.ts

```diff
@@ -29,8 +29,65 @@ export function revealInVSCode(revealable: IRevealable | undefined, omitFocus: b
     return Promise.resolve();
 }
 
-export function getApprovedTabs(callback: () => void) {
-    InspectorFrontendHost.InspectorFrontendHostInstance.getApprovedTabs(callback);
+export function getNetworkSetting(callback: () => void) {
+    // @ts-ignore we inject this function into the Runtime constructor where we call InspectorFrontendHost, not InspectorFrontendHost.InspectorFrontendHostInstance

```

Comment:
currently InspectorFrontendHost.InspectorFrontendHostInstance seems to be correctly typed. Why is the ts-ignore required in here?

Reply:
Fixed